src/mawpy/steps/incremental_clustering.py

```diff
@@ -69,17 +69,15 @@ def _merge_stays(stay_to_update, updated_stay, df_by_user, group_avgs, group_avg
     new_avg = merged_values.apply(_mean_ignore_minus_ones).fillna(-1)
 
     group_avgs.loc[group_avgs[STAY] == updated_stay, [STAY_LAT, STAY_LONG]] = new_avg.values
-    df_by_user.loc[df_by_user[STAY] == updated_stay, STAY_LAT] = new_avg.values[0]
-    df_by_user.loc[df_by_user[STAY] == updated_stay, STAY_LONG] = new_avg.values[1]
+    df_by_user.loc[df_by_user[STAY] == updated_stay, [STAY_LAT, STAY_LONG]] = new_avg.values
```

Comment:
`[STAY_LAT, STAY_LONG]` is repeated multiple times, can you create a new variable called `STAY_LAT_LONG`?

Reply:
updated.